about.html

```diff
@@ -0,0 +1,155 @@
+        <section class="border" id="logo">
+            <h2 class="logo-text text-center py-5">The National Educational Summit<br> Logo 2023</h2>
+            <div class="br-line"></div>
+            <h2 class="about-contact text-center mx-3">The logo of National Educational Summit was decided<br>through recently held logo competition</h2>
+            <div>
+                <img class="NES-logo pb-5" src="./images/NES+LOGO.png" alt="NES+LOGO">
+            </div>
+        </section>
+
+        <!-- Previous Educational Summits -->
+        <section id="past">
+            <h2 class="logo-text text-center pt-5">See the Past National Educational Summits</h2>
+            <div class="br-line"></div>
+            <h2 class="about-contact text-center mx-3">Take a Look at the last two National<br>Educational Summits</h2>
+            <div class="past-c">
+                <div class="past-1 mx-3 mt-5">
+                <h2 class="about-past0 text-center mx-3">2022</h2>
+                <h2 class="about-past1 text-center mx-3">National Educational Summit 2022, Washington,DC.</h2>
+                </div>
+                <div class="past-2 mx-3 mt-5">
+                    <h2 class="about-past3 text-center mx-3">2021</h2>
+                    <h2 class="about-past4 text-center mx-3">National Educational Summit 2021(online), Washington,DC.</h2>
+                </div>
+            </div>
+        </section>
+    </main>
+
+        <!-- Partner Section Goes There -->
+        <section class="partner-whole mt-5">
+            <h2 class="partners-title text-center pt-5">Partners</h2>
+            <div class="br-line"></div>
+            <div class="partners-main mx-5 text-center mt-5">
+               <div>
+                <img class="partners-pic" src="./images/Wellbeing+Web+(1).png" alt="">
+               </div> 
+               <div>
+                <img class="partners-pic" src="./images/Classroom+of+the+Future.png" alt="">
+               </div> 
+               <div>
+                <img class="partners-pic" src="./images/LIBRARIES+NEW+MILLENIUM.png" alt="">
+               </div> 
+               <div>
+                <img class="partners-pic" src="./images/Additional+Needs+Web+(1).png" alt="">
+               </div> 
+               <div>
+                <img class="partners-pic" src="./images/Literacy+Masterclass.png" alt="">
+               </div> 
+               <div>
+                <img class="partners-pic" src="./images/freeseminarsfloat.png" alt="">
+               </div>  
+            </div>
+        </section>
+
+        <!-- Footer goes there -->
+        <footer class="footer">
+            <h3 class="footer-1 text-center">
+                The National<br> Educational Summit
+            </h3>
+            <div>
+                <p class="footer-2 px-3">
+                    Originally Designed by Chindy Shin Behance, <br>some rights reserved 
+                </p>
+            </div>
+        </footer>
```

Comment:
- [ ] Please remove the "Partners" section from the "About" page to match the given design.
- [ ] Please adjust the background color of the footer section on the "About" page.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/225331866-0596920b-507a-46d2-a106-1b391ea8f0ae.png) | ![image](https://user-images.githubusercontent.com/62290545/225330727-19dc78c3-54aa-47a1-88ce-1194e6a230e7.png) |

Reply:
Definitely, This is what review for!Thanks Sir, for pointing out my mistakes